public/app/plugins/datasource/tempo/_importedDependencies/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains dependencies that we duplicated from Grafana core while working on the decoupling of Tempo from such core.
```

Comment:
Should we be duplicating dependencies? Can't we could move them from core into a yarn workspace and update both core import paths and the plugin import paths to point to a single source of truth?

Reply:
Ah sorry @fabrizio-grafana I hadn't noticed that. Fair enough, I'm not blocking anything and I understand you wanna get on. I do feel though that moving them around within this repo and leveraging yarn workspaces (as you're doing in the tempo-tracing package PR) would be a neater approach. It keeps a single source of truth and should make auditing and deciding where they should live easier once more and more plugins are extracted. Maybe I'm missing something though. 🤔